sources/updater/updog/src/bin/updata.rs

```diff
@@ -28,7 +28,7 @@ struct AddUpdateArgs {
     // metadata file to create/modify
     file: PathBuf,
 
-    // image 'variant', eg. 'aws-k8s-1.17'
+    // image 'variant', eg. 'aws-k8s-1.19'
```

Comment:
How about `aws-ecs-1` so we don't have to do this again?  :)

Same in these?
* `sources/logdog/build.rs`
* `sources/models/build.rs`
* `sources/api/early-boot-config/build.rs`
* `tools/pubsys/policies/ssm/README.md` (which you also have below)

Reply:
🤔

Reply:
Good idea, [pushed](https://github.com/bottlerocket-os/bottlerocket/compare/1781b6ac3afc70dfef80153bdf08bccf6cf16aa8..8dd6a341412cca3625aed8dfb79dc212a78b34b1)